assets/src/ba_data/python/bastd/activity/coopscorescreen.py

```diff
@@ -1193,22 +1193,22 @@ def _show_world_rank(self, offs_x: float) -> None:
             Text(ba.Lstr(resource='coopSelectWindow.timeRemainingText'),
                  position=(-360, -70 - 100),
                  color=(1, 1, 1, 0.7),
-                 h_align='center',
-                 v_align='center',
-                 transition='fade_in',
+                 h_align=Text.HAlign.CENTER,
+                 v_align=Text.VAlign.CENTER,
+                 transition=Text.Transition.FADE_IN,
                  scale=0.8,
                  maxwidth=300,
                  transition_delay=2.0).autoretain()
-            self._tournament_time_remaining_text = Text('',
-                                                        position=(-360,
-                                                                  -110 - 100),
-                                                        color=(1, 1, 1, 0.7),
-                                                        h_align='center',
-                                                        v_align='center',
-                                                        transition='fade_in',
-                                                        scale=1.6,
-                                                        maxwidth=150,
-                                                        transition_delay=2.0)
+            self._tournament_time_remaining_text = Text(
+                '',
+                position=(-360, -110 - 100),
+                color=(1, 1, 1, 0.7),
+                h_align=Text.VAlign.CENTER,
+                v_align=Text.VAlign.CENTER,
+                transition=Text.Transition.FADE_IN,
+                scale=1.6,
+                maxwidth=150,
+                transition_delay=2.0)
```

Comment:
This is an interesting case where type-checking wasn't running for these statements.
if turns out that the `if self._tournament_time_remaining is not None` line threw things off.
If I added a reveal_type() call above that statement, Mypy would give me a value, but adding one under the `if` statement gave nothing. This shows that Mypy wasn't evaluating that code.

It turns out that Mypy considered self._tournament_time_remaining to be None and so considered that code to be unreachable and thus didn't type check it. This is because self._tournament_type_remaining was declared simply as None. If I give it a proper Optional[float] type, Mypy does evaluate the code under the `if` statement and properly flags the error.